appveyor.yml

```diff
@@ -0,0 +1,58 @@
+# Based on the "trust" template v0.1.1
+# https://github.com/japaric/trust/tree/v0.1.1
+
+environment:
+  global:
+    RUST_VERSION: stable
```

Comment:
One other small inconsistency nit: This is `RUST_VERSION` but `.travis.yml` has this named `NIGHTLY` (and as such is a specific nightly version, while this is _latest stable_ [or am I wrong about that?]).

If this is _latest stable_ ideally we would pin to a particular version.

Reply:
Travis uses stable for all stages except `lint`, where we explicitly state `NIGHTLY` in `rust:` and `cargo +$NIGHTLY` statements. Maybe I should have worded the global var as `NIGHTLY_VERS` to make it less confusing.

This `RUST_VERSION` var in appveyor is used later in rustup scripts so it does decide what version of rust we use in appveyor, unlike travis which defaults to stable unless told otherwise.